test/e2e/backup-restore/backup_restore_test.go

```diff
@@ -126,7 +124,7 @@ func testBackupRestore(t *testing.T, clusterSpec clusterSpec, clusterSize int) {
 	sourceCluster := name + "-source"
 	infinispan := clusterSpec(sourceCluster, namespace, clusterSize)
 	testKube.Create(infinispan)
-	defer testKube.DeleteInfinispan(infinispan, tutils.SinglePodTimeout)
+	defer testKube.CleanNamespaceAndLogOnPanic(namespace)
```

Comment:
Is there a way to disable cleanup via flag/property  on failure for debugging purposes?

Reply:
I have updated the code now, so it would just be a case of commenting the `DeleteAllOf` commands in `CleanNamespaceAndLogOnPanic` .. if you think it would be useful, we can add a env var to toggle this. Any thoughts on what it should be called?

I'm just investigation why backup/restore is not  playing nice with travis atm, but my inteion is to make use of this method in all tests.

Reply:
Maybe `KEEP_DEPLOYMENT_RUNNING`?

Reply:
`CLEANUP_INFINISPAN_ON_FINISH` already exists, so we can just reuse that.